src/libstd/lib.rs

```diff
@@ -299,6 +299,7 @@
 #![feature(unwind_attributes)]
 #![feature(vec_push_all)]
 #![feature(zero_one)]
+#![cfg_attr(target_os = "redox", feature(naked_functions))]
```

Comment:
#### Hygiene error! In order to generate executables, we have to have a naked `_start` function at the libstd root.